core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -28,20 +26,16 @@
 public final class TenantConstants extends RequestResponseApiConstants {
 
     /* tenant actions */
-    public static final String ACTION_GET                        = "get";
-    public static final String ACTION_ADD                        = "add";
-    public static final String ACTION_UPDATE                     = "update";
-    public static final String ACTION_REMOVE                     = "remove";
-
+    public enum ACTIONS {
```

Comment:
This is not a valid class/enum name FMPOV, can we change it to `Action` or `Operation`?